### PR TITLE
make acceptType check case insensitive

### PR DESCRIPTION
**Pull request by masesisaac**
Solves issue #60

### CODE REVIEW

**Comment by masesisaac** [comment]
@vutoan266 Thanks for this wonderful library.

Please let me know in case of any recommended changes to this PR.

File: src/validation.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const isMaxFileSizeValid = (fileSize, maxFileSize) => {
 export const isAcceptTypeValid = (acceptType, fileName) => {
   if (acceptType && acceptType.length > 0) {
     const type: string = fileName.name.split('.').pop() || '';
-    if (acceptType.indexOf(type) < 0) {
+    if (acceptType.findIndex(item => item.toLowerCase() === type.toLowerCase()) < 0) {
```
Comment (davidnguyen179) [inline src/validation.ts:67]: why don't you use "includes" method?

**Comment by davidnguyen179** [comment]
looks good

**Review by vutoan266 — APPROVED** [review]

**Comment by vutoan266** [comment]
@masesisaac Thanks for your contribution 👍

**Comment by davidnguyen179** [comment]
@all-contributors please add @masesisaac for code

**Pull request merged**